runtime/oti/jvminit.h

```diff
@@ -682,6 +682,8 @@ enum INIT_STAGE {
 
 #define VMOPT_XTRACE "-Xtrace"
 
+#define VMOPT_XDYNAMICHEAPADJUSTMENTFORRESTORE "-XdynamicHeapAdjustmentForRestore"
```

Comment:
since this is really used once, we don't really need a #define - just use the string directly